### ISSUE TITLE
Minimal example for JSON logging: Is this okay?

### ISSUE THREAD

**Issue by windymelt** [outr/scribe#564]
At first, thanks for great logging library! I'm going to replace existing logging framework to Scribe. :)

However, I'm troubled in using Scribe as JSON logger, because there is few documentation for JSON logging on current version.

I wrote these code to use JSON logging, but I'm worried whether this is okay or not.

## Code I wrote

At first we need JSON binding:

```scala
libraryDependencies += "com.outr" %% "scribe-json-circe" % "3.13.0"
```

```scala
import scribe.json.ScribeCirceJsonSupport
scribe.Logger.root
  .orphan()
  .clearHandlers()
  .withHandler(
    writer = ScribeCirceJsonSupport.writer(scribe.writer.ConsoleWriter)
  )
  .replace()

scribe.info("Hello, JSON!")
```

Is this okay enough?

## Log I get (expanded for readability)

```json
{
  "messages": [
    "Hello, JSON!"
  ],
  "service": null,
  "level": "INFO",
  "value": 300,
  "fileName": "***.scala",
  "className": "foo.bar",
  "methodName": "foobarMEthod",
  "line": 42,
  "thread": "brah-brah-thread",
  "@timestamp": "2024-03-19T17:29:43.997Z",
  "stack_trace": null,
  "mdc": {},
  "data": {}
}
```

## Additional question

Can I extend JSON with extra field? for example `severity` (I want to just copy `level` field), `logging.googleapis.com/trace` (for Google Cloud Logging / Cloud Trace)

**Comment by darkfrog26** [comment]
Released with 3.13.1

**Comment by windymelt** [comment]
Thank you from the bottom of my heart!!!
